conda_forge_tick/auto_tick.xsh

```diff
@@ -186,8 +186,7 @@ def main(args=None):
                     if is_github_api_limit_reached(e, gh):
                         break
             except Exception as e:
-                logger.critical('NON GITHUB ERROR')
-                logger.critical(str(e))
+                logger.exception('NON GITHUB ERROR')
```

Comment:
aren't you missing the actual error message here?

Reply:
My understanding of the `exception` method is that it includes the traceback in the logger message.

Reply:
Will this also include the standard error for CLI tools?

Reply:
Oh you are right!